src/app/hooks/useEffectOnce.ts

```diff
@@ -4,6 +4,7 @@ export const useEffectOnce = (effect: () => void | (() => void)) => {
   const destroyFunc = useRef<void | (() => void)>();
   const effectCalled = useRef(false);
   const renderAfterCalled = useRef(false);
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const [_, setVal] = useState<number>(0);
```

Comment:
No need to disable eslint here, if I understand correctly, you want to skip this -> _
You can do it like that:
const [, setVal] = useState<number>(0);

Reply:
thank you @Chriserus , if you have any remarque or a better way to do things let me know ^^